ambry-store/src/main/java/com.github.ambry.store/StoreFindToken.java

```diff
@@ -52,23 +54,29 @@
   private final Offset offset;
   private final StoreKey storeKey;
   private final UUID sessionId;
+  private final UUID incarnationId;
   private long bytesRead;
 
   StoreFindToken() {
```

Comment:
Can/Will this constructor be gotten rid of once Version_0 and Version_1 are phased out?